### PR TITLE
refactor: move json utility function to common crate

### LINKED ISSUE #57: move the json utils inside the common crate
we have a common crate that is built for sharing common implementation through other cartes, the `json_utils::init_payload()` inside the plugin should be moved into the common crate

### PR DESCRIPTION
**Pull request by swaptr**
Move the `json_utils::init_payload()` to the common crate for cogent access.
Fixes #57

### CODE REVIEW

File: common/src/payload.rs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+//! This file provides utility functions to initialize an empty payload for a JSON response.
+//!
+//! author: https://github.com/vincenzopalazzo
+use serde_json::json;
+
+/// return a server::json_value useful to initialize the JSON payload as an empty JSON object.
+pub fn init_payload() -> serde_json::Value {
```
Comment (vincenzopalazzo) [inline common/src/payload.rs:7]: better keep the name of the file `json_utils` but I think you can just copy and paste it?

**Review by vincenzopalazzo — CHANGES_REQUESTED** [review]

File: common/src/payload.rs (hunk 1 above)
Comment (swaptr) [reply common/src/payload.rs]: You are right! Fixed.
Thanks!

File: plugin/src/commands/builtin.rs [hunk 2]
```diff
@@ -6,13 +6,12 @@ use crate::commands::{
...
     RPCCommand,
 };
-// FIXME: move this inside the common crater
-use crate::commands::json_utils::{add_bool, add_vec, init_payload};
 use crate::errors::PluginError;
 use crate::plugin::{OnInit, Plugin};
 use crate::types::RpcOption;
 use serde_json::Value;
 
+use clightningrpc_common::json_utils::{add_bool, add_vec, init_payload};
```
Comment (vincenzopalazzo) [inline plugin/src/commands/builtin.rs:14]:
```suggestion
use serde_json::Value;
use clightningrpc_common::json_utils::{add_bool, add_vec, init_payload};

```

**Review by vincenzopalazzo — APPROVED** [review]
Thanks! a small fix in the formatting of an import and also the annoying fix of the commit message :)

`fix(common): move the json_utils inside from plugin to common crate`

**Review by vincenzopalazzo — APPROVED** [review]
As always great Job! Thanks

**Pull request merged**